pcc-framework.php

```diff
@@ -9,7 +9,7 @@
  * Domain Path:     /languages
  * License:         BSD 3-Clause "New" License
  * License URI:     https://opensource.org/licenses/BSD-3-Clause
- * Version:         2.2.2
+ * Version:         2.2.1
```

Comment:
This is a change that snuck in from a different commit. Should this get reverted?

Reply:
Nope, I released 2.2.2 recently so that's as expected (I think you branched from dev before that release).